### PR TITLE
Updates from SfB to MS Teams #3140

### PR DESCRIPTION
**Pull request by j0rt3g4**
Updates from SfB to MS Teams #3140

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys.
Thank you in advance!

File: Teams/here-s-what-you-get-with-phone-system.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ description: "Learn about the features, availability, and how to plan and set up
 
 A PBX is a phone system within a business. The Phone System in Office 365 add-on gives you PBX capabilities, but without the complicated and expensive equipment. 
   
-With the Phone System add-on, you can use Skype for Business to complete basic tasks such as making and receiving calls, transferring calls, and muting or unmuting calls, from nearly anywhere with Internet access. It's available worldwide! 
+With the Phone System add-on, you can use Skype for Business or Microsoft Teams to complete basic tasks such as making and receiving calls, transferring calls, and muting or unmuting calls, from nearly anywhere with Internet access. It's available worldwide!
```
Comment (JohanFreelancer9) [inline Teams/here-s-what-you-get-with-phone-system.md]:
```suggestion
With the Phone System add-on, you can use Skype for Business or Microsoft Teams to complete basic tasks such as making and receiving calls, transferring calls, and muting or unmuting calls from nearly anywhere with Internet access. It's available worldwide! 
```

File: Teams/here-s-what-you-get-with-phone-system.md [hunk 2]
```diff
@@ -40,25 +40,24 @@ By using Phone System, your Skype for Business users will get the following feat
...
-|[Call forwarding and simultaneously ring](https://support.office.com/article/967d9aaf-3fed-448b-ab96-40bbc9a11a20) <br/> |These features allow you to set up forwarding rules so your calls can go with you anywhere, and you can forward calls to colleagues or to voicemail.  <br/> |
-|[Find a previous conversation - Lync](https://support.office.com/article/d1d54123-0301-482e-b047-c82e9e2724a3) <br/> |Keep track of all your conversations in one place, whether those conversations are from IMs, phone calls, or impromptu and scheduled meetings. Conversations are recorded in your call history.  <br/> |
+|[Set Call Forwarding options & simultaneously ring for Microsoft Teams](https://support.office.com/article/call-forwarding-call-groups-and-simultaneous-ring-in-teams-a88da9e8-1343-4d3c-9bda-4b9615e4183e) <br/> |Make or answer calls on behalf of a manager you support. Notifications make it clear to all participants when calls are being answered or made for someone else. Also These features allow you to set up forwarding rules so your calls can go with you anywhere, and you can forward calls to colleagues or to voicemail <br/> |
```
Comment (JohanFreelancer9) [inline Teams/here-s-what-you-get-with-phone-system.md]:
```suggestion
|[Set Call Forwarding options and simultaneously ring in Microsoft Teams](https://support.office.com/article/call-forwarding-call-groups-and-simultaneous-ring-in-teams-a88da9e8-1343-4d3c-9bda-4b9615e4183e) <br/> |Make or answer calls on behalf of a manager you support. Notifications make it clear to all participants when calls are being answered or made for someone else. These features also allow you to set up forwarding rules so your calls can go with you anywhere, and you can forward calls to colleagues or to voicemail <br/> |
```

File: Teams/here-s-what-you-get-with-phone-system.md [hunk 3]
```diff
@@ -40,25 +40,24 @@ By using Phone System, your Skype for Business users will get the following feat
...
+|[Find a previous conversation - MS Teams](https://support.office.com/article/resume-an-old-conversation-in-teams-ab9f99a1-65a6-47b8-ba8a-79aa7352977d) <br/> |Keep track of all your conversations in one place, whether those conversations are from IMs, phone calls, or impromptu and scheduled meetings. Conversations are recorded in your call history.  <br/> |
```
Comment (JohanFreelancer9) [inline Teams/here-s-what-you-get-with-phone-system.md]:
```suggestion
|[Find a previous conversation - Microsoft Teams](https://support.office.com/article/resume-an-old-conversation-in-teams-ab9f99a1-65a6-47b8-ba8a-79aa7352977d) <br/> |Keep track of all your conversations in one place, whether those conversations are from IMs, phone calls, or impromptu and scheduled meetings. Conversations are recorded in your call history.  <br/> |
```

File: Teams/here-s-what-you-get-with-phone-system.md [hunk 4]
```diff
@@ -40,25 +40,24 @@ By using Phone System, your Skype for Business users will get the following feat
...
 |Call hold/retrieve  <br/> | Use when multiple calls occur at the same time. When you answer the next inbound call or place an outbound call, your current call goes on hold automatically. <br/> |
-|[Transfer a Skype for Business call](https://support.office.com/article/f9f312a3-ccc9-4215-86e8-8928b2df1f97) <br/> |Transfers calls to another person. Or, if you need to leave your office but want to continue your conversation, you can transfer the calls from your PC or IP phone to your cell phone or tablet.  <br/> |
+|[Transfer a Microsoft Teams call](https://support.office.com/en-us/article/transfer-a-call-in-teams-b7f40f14-e083-46b9-b739-68038c8f73a0) <br/> |Transfers calls to another person. Or, if you need to leave your office but want to continue your conversation, you can transfer the calls from your PC or IP phone to your cell phone or tablet.  <br/> |
```
Comment (JohanFreelancer9) [inline Teams/here-s-what-you-get-with-phone-system.md]:
```suggestion
|[Transfer a Microsoft Teams call](https://support.office.com/article/transfer-a-call-in-teams-b7f40f14-e083-46b9-b739-68038c8f73a0) <br/> |Transfers calls to another person. Or, if you need to leave your office but want to continue your conversation, you can transfer the calls from your PC or IP phone to your cell phone or tablet.  <br/> |
```

File: Teams/here-s-what-you-get-with-phone-system.md [hunk 5]
```diff
@@ -40,25 +40,24 @@ By using Phone System, your Skype for Business users will get the following feat
...
 |Clients for PC, Mac, and Mobile  <br/> | This gives you calling features on devices, from tablets and mobile phones to PCs and desktop IP phones. <br/> |
 |Device switching  <br/> |Enables you to play your call or meeting on another device.  <br/> |
-|[Change your Lync alerts](https://support.office.com/article/15DB9F71-A3D1-4AD7-9D84-39F05520B4C2) <br/> |Play different ringtones for the different types of calls you get every day, so you quickly know who is calling you.  <br/> |
-|[Change your presence status in Skype for Business](https://support.office.com/article/9b64eef5-47b8-46d8-a744-f490e6f88feb) <br/> |Controls your inbound communications with presence, enabling you to block all incoming communication except from those you specifically indicate.  <br/> |
+|[Change your notifications in Teams](https://support.office.com/article/manage-notifications-in-teams-1cc31834-5fe5-412b-8edb-43fecc78413d) <br/> |Play different ringtones for the different types of calls you get every day, so you quickly know who is calling you.  <br/> |
```
Comment (JohanFreelancer9) [inline Teams/here-s-what-you-get-with-phone-system.md]:
```suggestion
|[Change your notifications in Microsoft Teams](https://support.office.com/article/manage-notifications-in-teams-1cc31834-5fe5-412b-8edb-43fecc78413d) <br/> |Play different ringtones for the different types of calls you get every day, so you quickly know who is calling you.  <br/> |
```

File: Teams/here-s-what-you-get-with-phone-system.md [hunk 6]
```diff
@@ -40,25 +40,24 @@ By using Phone System, your Skype for Business users will get the following feat
...
+|[Check you voicemail in MS Teams](https://support.office.com/en-gb/article/check-your-voicemail-in-teams-f8d568ce-7329-4fe2-a6a2-325ec2e2b419) <br/> |Here's how Cloud Voicemail works: When a user receives a voicemail, it is delivered to their mailbox as an email with the voicemail message as an attachment. They can also listen to their messages over their Skype certified desktop phone, all Skype for Business applications, or Lync for Mac 2011.  <br/> Cloud Voicemail includes personalized greeting, message waiting indicator, and reply with call. You can have all of your voicemail deposited in mailboxes and made available through Skype for Business on your PC, mobile device, and IP phone.  <br/> |
```
Comment (JohanFreelancer9) [inline Teams/here-s-what-you-get-with-phone-system.md]:
```suggestion
|[Check you voicemail in Microsoft Teams](https://support.office.com/en-gb/article/check-your-voicemail-in-teams-f8d568ce-7329-4fe2-a6a2-325ec2e2b419) <br/> |Here's how Cloud Voicemail works: When a user receives a voicemail, it is delivered to their mailbox as an email with the voicemail message as an attachment. They can also listen to their messages over their Skype certified desktop phone, all Skype for Business applications, or Lync for Mac 2011.  <br/> Cloud Voicemail includes personalized greeting, message waiting indicator, and reply with call. You can have all of your voicemail deposited in mailboxes and made available through Skype for Business on your PC, mobile device, and IP phone.  <br/> |
```

File: Teams/here-s-what-you-get-with-phone-system.md [hunk 7]
```diff
@@ -70,22 +69,23 @@ Set up Phone System in the way that best suits your business:
...
+- You can get new phone numbers from Office 365, or transfer existing phone numbers to Skype for Business or Microsoft Teams. 
     
 See these articles to plan and set up the Phone System in your business:
 
 - [Phone System with Calling Plans](calling-plan-landing-page.md)
 
 - [Set up Calling Plans](set-up-calling-plans.md): This article has step-by-step instructions to get set up with your phone numbers or to transfer your existing phone numbers to Skype for Business.
     
-- [Microsoft telephony solutions](https://docs.microsoft.com/SkypeForBusiness/hybrid/msft-telephony-solutions): This  article lays out all of your deployment options. It also has information about planning an entire cloud-only solution. 
+- [Microsoft Teams solutions](https://www.microsoft.com/microsoft-365/partners/teams-custom-solutions): This  article lays out on information about planning an entire cloud-only solution.
```
Comment (JohanFreelancer9) [inline Teams/here-s-what-you-get-with-phone-system.md]:
```suggestion
- [Microsoft Teams solutions](https://www.microsoft.com/microsoft-365/partners/teams-custom-solutions): This  article lays out all information about planning an entire cloud-only solution. 
```

**Comment by j0rt3g4** [comment]
Thank you @JohanFreelancer9

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
Thanks for your feedback.  We've received numerous comments on this topic and updates are in progress.  I want to consolidate all the feedback before we merge any pull requests.  Look for an update soon.

**Comment by CarolynRowe** [comment]
This topic has been updated.  Closing this PR.

**Pull request closed without merging**